src/common/components/card/LargeCollectionCard.tsx

```diff
@@ -16,7 +17,10 @@ function LargeCollectionCard({
     <Card id={id} className={styles.largeCollectionCard}>
       <Card.Content className={styles.text}>
         <Card.Title as="h3" variant="h2" title={title} href={href} />
-        <Card.InfoLines variant="body-l" infoLines={infoLines} />
+        <Card.ClampedCardInfoLines
+          variant="body-l"
+          infoLines={[formatToSingleTag(infoLines[0], "p")]}
```

Comment:
On this line we are taking the first member of the array. I wonder if that's the best API. I wonder if it'd be clearer for the dev using this component if we looped over all the lines so that we don't accidentally lose any of them.

E.g. "pseudo code"

```ts
lines.map((line, i) => i === 0 ? formatToSingleTag("text" in line ? line.text : line) : line)
```

You aded `p` as the default value for the second parameter, so you no longer need to declare it here.